src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -79,38 +222,126 @@ export default function InfoMultipleSection() {
+                changeStationsTypeSelected(graph.current, selected, newStationType);
+                setNewStationType(undefined);
+                hardRefresh();
+            }
+        }
+        setIsChangeTypeWarningOpen(false);
+    };
+
     return (
-        <Box>
-            <Heading as="h5" size="sm">
-                {t('panel.details.selected')} {selected.size}
-            </Heading>
-            <RmgLabel label={t('panel.details.multipleChangeColor')}>
-                <ThemeButton
-                    theme={theme}
-                    onClick={() => {
-                        setIsThemeRequested(true);
-                        dispatch(openPaletteAppClip(theme));
-                    }}
-                />
-            </RmgLabel>
-            <VStack m="var(--chakra-space-1)">
-                {[...selected].map(id => (
-                    <Button
-                        key={id}
-                        width="100%"
-                        size="sm"
-                        variant="solid"
-                        onClick={() => dispatch(setSelected(new Set([id])))}
-                        overflow="hidden"
-                        maxW="270"
-                        textOverflow="ellipsis"
-                        whiteSpace="nowrap"
-                        display="ruby"
-                    >
-                        {getName(id)?.replaceAll('\\', '⏎')}
+        <Modal isOpen={isOpen} onClose={onClose} size="sm" scrollBehavior="inside">
+            <ModalOverlay />
+            <ModalContent>
+                <ModalHeader>
+                    <Text as="b" fontSize="xl">
+                        {t('panel.details.multipleSelection.change')}
+                    </Text>
+                </ModalHeader>
+
+                <ModalBody>
+                    {filterNodes && (
+                        <RmgLabel label={t('panel.details.multipleSelection.changeStationType')} minW="276">
+                            <RmgSelect
+                                options={availableStationOptions}
+                                defaultValue="default"
+                                value="default"
+                                onChange={({ target: { value } }) => {
+                                    setNewStationType(value as StationType);
+                                    setIsChangeTypeWarningOpen(true);
+                                }}
+                            />
+                        </RmgLabel>
+                    )}
+                    {filterEdges && (
+                        <>
+                            <RmgLabel label={t('panel.details.multipleSelection.changeLinePathType')} minW="276">
+                                <RmgSelect
+                                    options={availableLinePathOptions}
+                                    disabledOptions={['simple']}
+                                    defaultValue="default"
+                                    value="default"
```

Comment:
This is weird as the value never changes. How about give two selection: the station type your want to change from and the station type you want to change to. And then have another button to perform the change action.